File: BackEnd/Service/Models/UserModel.cs
namespace Service.Models
{
    public class UserModel
    {
        public string Id { get; set; }

        public string? FullName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string? Address { get; set; }
        public string? ImageURL { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public Guid? CandidateId { get; set; }
        public Guid? InterviewerId { get; set; }
        public Guid? RecruiterId { get; set; }
        public Guid? CompanyId { get; set; }
    }
}

File: FRONTEND/src/app/data/user/user.model.ts
export class User {
	id?: string;
	fullName?: string;
	dateOfBirth?: Date;
	address?: string;
	imageUrl?: string;
	userName?: string;
	normalizedUserName?: string;
	email?: string;
	normalizedEmail?: string;
	emailConfirmed?: boolean;
	passwordHash?: string;
	securityStamp?: string;
	phoneNumber?: string;
	phoneNumberConfirmed?: boolean;
	twoFactorEnabled?: boolean;
	lockoutEnd?: Date;
	lockoutEnabled?: boolean;
}
